products/casp/main.pm

```diff
@@ -32,26 +32,42 @@ my $distri = testapi::get_var("CASEDIR") . '/lib/susedistribution.pm';
 require $distri;
```

Comment:
Guys, if you want to change how this is done, then I believe it should not block this PR. The PR itself does not change the initialization itself.